ntoskrnl/cc/view.c

```diff
@@ -75,13 +75,16 @@ ULONG CcRosVacbIncRefCount_(PROS_VACB vacb, PCSTR file, INT line)
 ULONG CcRosVacbDecRefCount_(PROS_VACB vacb, PCSTR file, INT line)
 {
     ULONG Refs;
+    BOOLEAN VacbDirty = vacb->Dirty;
+    BOOLEAN VacbTrace = vacb->SharedCacheMap->Trace;
+    BOOLEAN VacbPageOut = vacb->PageOut;
 
     Refs = InterlockedDecrement((PLONG)&vacb->ReferenceCount);
```

Comment:
In ntoskrnl code (only) we have the InterlockedIncrementUL and InterlockedDecrementUL helpers that @tkreuzer could use here in order to avoid all these extra casts.

Reply:
I could, but this PR is not to fix coding style but to fix actual bugs.